book/intro.md

```diff
@@ -1,11 +1,89 @@
-# Welcome to your Jupyter Book
+# 🥳 Welcome to the EGU22 PyGMT short course
 
-This is a small sample book to give you a feel for how book content is
-structured.
-It shows off a few of the major file types, as well as some sample content.
-It does not go in-depth into any particular topic - check out [the Jupyter Book documentation](https://jupyterbook.org) for more information.
+This Jupyter book contains [PyGMT](https://www.pygmt.org/v0.6.1) tutorials
+for producing maps and doing geospatial data processing 🌍
 
-Check out the content pages bundled with this sample book to see more.
+- [Anatomy of a PyGMT figure](first-figure) by [Leonardo Uieda](https://orcid.org/0000-0001-6123-9515)
+- [Integration with the scientific Python ecosystem 🐍](ecosystem) by [Max Jones](https://orcid.org/0000-0003-0180-8928)
+- [Making some Mars maps with pygmt](Mars_Maps) by [André Luiz Belém](https://orcid.org/0000-0002-8865-6180)
```

Comment:
Placeholder for André's tutorial. Will update the link once the notebook is in this repo.

Oh, and do you think it's ok to put it here between the ecosystem and LiDAR tutorials? Or move it to another position?

Reply:
Yes, I think this is a good spot.